packages/prelude/test/function-test.js

```diff
@@ -12,6 +12,20 @@ describe('id', () => {
   })
 })
 
+describe('pipe', () => {
+  it('pipe(f, g)(x) === f(g(x))', () => {
+    const fx = 'fx'
+    const gx = 'gx'
+    const x = 'x'
```

Comment:
Please match the style of the tests below by using `Math.random`, or update the tests below to match the style of this one.  Either is ok with me.